node/package-lock.json

```diff
@@ -0,0 +1,2527 @@
+{
+  "name": "git-meta",
```

Comment:
Do you really want to commit the lock file? Feels like it was intentionally not committed earlier.

Reply:
Yes, according to the [official doc](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md) of `package-lock.json`, it is intended to be checked in:

> package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.
> This file is intended to be committed into source repositories, and serves various purposes: